modules/ROOT/pages/_partials/attributes.adoc

```diff
@@ -37,8 +37,8 @@
 :mobilecicd-ui: Jenkins UI
 
 // Device Security
-:device-security-service: Device Security
-:app-security-service: App Security
+:device-security-service: Device Checks
+:app-security-service: Mobile Security
```

Comment:
`Mobile Security` seems a bit vague to me

/2cents

Reply:
prob not a great place to review that, cos in the docs it typically appears as 'Mobile Security Service'
and that name comes from Product/Eng, so no option to change it...
in fact, changing it to this value is the whole purpose of this PR.

Reply:
Ah got ya - for some reason I skimmed quickly over this area and thought it was just an internal heading name we were using :) 

Anyway, agreed :) :+1: